packages/storybook/cypress/support/commands.js

```diff
@@ -23,3 +23,79 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('selection', { prevSubject: true }, (subject, fn) => {
+    cy.wrap(subject)
+      .trigger('mousedown')
+      .then(fn)
+      .trigger('mouseup');
+  
+    cy.document().trigger('selectionchange');
+    return cy.wrap(subject);
+  });
+  
+  Cypress.Commands.add('setSelection', { prevSubject: true }, (subject, query, endQuery) => {
+    return cy.wrap(subject)
+      .selection($el => {
```

Comment:
@irmerk Yes you are on the right path.Here the `$el` should identify the text as node although I have wrapped a node here in `selection` custom command.But still it is taking up the text rather than the element.
Plus @sanketshevkar PR #311 would also fail until we found a way to select the text.

Reply:
#311 does require the text to be selected, its about placing the cursor in a para and then changing the headings.

Reply:
I think I'm not following after all. Why do we need these custom commands when Slate has built in functions for this?

Reply:
@irmerk I guess you are not on the right track.Since slate has no connection with the selection of commands.The Cypress automates that work which a user has to do.The selection of text must be done by Cypress.